app/components/emailValidation/CorrespondingAuthorMistake.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { Flex, Box } from 'grid-styled'
+import { Heading, Button } from '@pubsweet/ui'
+
+const CorrespondingAuthorMistake = () => (
+  <Flex>
+    <Box>
+      <Heading level={1}>Thank you!</Heading>
+      <p>You will not be listed as the corresponding author for [title].</p>
+      <Button>Made a mistake?</Button>
```

Comment:
Should this link the confirmation page?

Reply:
Yeah this page is only placeholder atm. Just so I could see that clicking 'Decline' from within the email would take me to this page. Haven't linked up 'Made a mistake?' to anything.

Wasn't quite sure from the product requirements, where this button should take a user. Why would 'Made a mistake?' take them to the confirmation page?

Reply:
'Made a mistake' is if they have accidentally used the 'Decline' link within the email (i.e. they want to undo this action and actually confirm they are the corresponding author... hence being redirected to the confirmation page).

Reply:
Oh! Gotcha, thanks :+1: 

Perhaps it's just me, but I didn't recognise that from the wording. I'm wondering whether there's any way we can make it more obvious to users, what's going to happen if they click on these links?

Since it says categorically 'You **will not** be listed...', I wouldn't necessarily expect to click a button saying 'Made a mistake?' and have that automatically reverse my original decision.

Happy to implement what's been given, obviously. Just food for thought :smile: